src/stories/theme/tokens.ts

```diff
@@ -1,4 +1,75 @@
 export const Colors = {
+    new: {
```

Comment:
just to keep types simpler for the transitory state, I would restructure this a little bit to match the structure of theme — instead of `new` living inside colors, it should live inside the parent `tokens`

so you could do something like `export const ColorsNew = { ... }` with all of the new colors, and then at the bottom, have the `TokensT` export be 
```ts
export type TokensT = {
    colors: typeof Colors,
    new: {
        colors: typeof ColorsNew,
    },
};
```

and of course then update the exported object accordingly. this allows for better typing per my comment on the themes file, where we can now use `TokensT['new']` to reference the upcoming new tokens type for a clean swap later